Localizations/en_US.json

```diff
@@ -148,13 +148,13 @@
     "script_content_level_1": "Level 1",
     "script_content_level_1_description": "Methods are stripped from decompiled export.",
     "script_content_level_2": "Level 2",
-    "script_content_level_2_description": "Default. This exports full methods for Mono games and dummy methods for IL2Cpp games.",
+    "script_content_level_2_description": "Default. This exports full methods for Mono games and dummy methods for IL2CPP games.",
     "script_content_level_3": "Level 3",
-    "script_content_level_3_description": "IL2Cpp methods are safely recovered where possible.",
+    "script_content_level_3_description": "IL2CPP methods are safely recovered where possible.",
     "script_content_level_4": "Level 4",
-    "script_content_level_4_description": "IL2Cpp methods are recovered without regard to safety.",
+    "script_content_level_4_description": "IL2CPP methods are recovered without regard to safety.",
     "script_content_level_title": "Script Content Level",
-    "script_export_format_decompiled": "Decompiled",
+    "script_export_format_decompiled": "Decompile",
```

Comment:
This is a linguistic decision. Ideally, all 4 options would be the same type of word. In this context, "Export" is a noun, and "Hybrid" can be either a noun or an adjective. If we're changing "Decompiled," it should become a noun like the others. It should be "Decompilation," and the Shader option should be changed, too.

Reply:
Oh, I hadn't noticed that! My bad! Anyway, I'll change the word to "Decompilation."